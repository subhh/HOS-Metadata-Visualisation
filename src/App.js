import React, { useEffect, useReducer, useCallback } from 'react'
import { Route, matchPath } from 'react-router-dom'
import { initialState, StateContext, DispatchContext, reducer } from './state'
import { updateQuery } from './state/actions'
import Stats from './components/Stats'
import Layout from './components/Layout'
import HeroContainer from './components/HeroContainer'

export const AppWithoutProvider = () => {
  const renderHero = useCallback(
    ({ width, height, targetHeight }) => (
      <HeroContainer
        width={width}
        height={height}
        targetHeight={targetHeight}
      />
    ),
    []
  )

  return (
    <Route
      render={({ match }) => {
        const expand = matchPath(match.path, { path: '*/explore' })
        return (
          <Layout
            expand={expand}
            renderHero={renderHero}
            renderContent={() => <Stats />}
          />
        )
      }}
    />
  )
}

const App = ({ query }) => {
  const [state, dispatch] = useReducer(reducer, initialState)

  useEffect(() => {
    dispatch(updateQuery(query))
  }, [query])

  return (
    <StateContext.Provider value={state}>
      <DispatchContext.Provider value={dispatch}>
        <AppWithoutProvider />
      </DispatchContext.Provider>
    </StateContext.Provider>
  )
}

export default App
