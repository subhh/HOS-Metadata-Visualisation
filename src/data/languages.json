[
  {
    "name": "Afar-Sprache",
    "code": "aa"
  },
  {
    "name": "Abchasisch",
    "code": "ab"
  },
  {
    "name": "Awestisch",
    "code": "ae"
  },
  {
    "name": "Afrikaans",
    "code": "af"
  },
  {
    "name": "Akan-Sprache",
    "code": "ak"
  },
  {
    "name": "Amharisch",
    "code": "am"
  },
  {
    "name": "Aragonesisch",
    "code": "an"
  },
  {
    "name": "Arabisch",
    "code": "ar"
  },
  {
    "name": "Assamesisch",
    "code": "as"
  },
  {
    "name": "Awarisch",
    "code": "av"
  },
  {
    "name": "Aymara-Sprache",
    "code": "ay"
  },
  {
    "name": "Aserbeidschanisch",
    "code": "az"
  },
  {
    "name": "Baschkirisch",
    "code": "ba"
  },
  {
    "name": "Weißrussisch",
    "code": "be"
  },
  {
    "name": "Bulgarisch",
    "code": "bg"
  },
  {
    "name": "Bihari",
    "code": "bh"
  },
  {
    "name": "Bislama; Beach-la-mar",
    "code": "bi"
  },
  {
    "name": "Bambara-Sprache",
    "code": "bm"
  },
  {
    "name": "Bengali",
    "code": "bn"
  },
  {
    "name": "Tibetisch",
    "code": "bo"
  },
  {
    "name": "Bretonisch",
    "code": "br"
  },
  {
    "name": "Bosnisch",
    "code": "bs"
  },
  {
    "name": "Katalanisch",
    "code": "ca"
  },
  {
    "name": "Katalanisch",
    "code": "ca"
  },
  {
    "name": "Tschetschenisch",
    "code": "ce"
  },
  {
    "name": "Chamorro-Sprache",
    "code": "ch"
  },
  {
    "name": "Korsisch",
    "code": "co"
  },
  {
    "name": "Cree-Sprache",
    "code": "cr"
  },
  {
    "name": "Tschechisch",
    "code": "cs"
  },
  {
    "name": "Kirchenslawisch",
    "code": "cu"
  },
  {
    "name": "Kirchenslawisch",
    "code": "cu"
  },
  {
    "name": "Tschuwaschisch",
    "code": "cv"
  },
  {
    "name": "Kymrisch",
    "code": "cy"
  },
  {
    "name": "Dänisch",
    "code": "da"
  },
  {
    "name": "Deutsch",
    "code": "de"
  },
  {
    "name": "Maledivisch",
    "code": "dv"
  },
  {
    "name": "Dzongkha",
    "code": "dz"
  },
  {
    "name": "Ewe-Sprache",
    "code": "ee"
  },
  {
    "name": "Griechisch",
    "code": "el"
  },
  {
    "name": "Englisch",
    "code": "en"
  },
  {
    "name": "Esperanto",
    "code": "eo"
  },
  {
    "name": "Spanisch",
    "code": "es"
  },
  {
    "name": "Spanisch",
    "code": "es"
  },
  {
    "name": "Estnisch",
    "code": "et"
  },
  {
    "name": "Baskisch",
    "code": "eu"
  },
  {
    "name": "Persisch",
    "code": "fa"
  },
  {
    "name": "Ful",
    "code": "ff"
  },
  {
    "name": "Finnisch",
    "code": "fi"
  },
  {
    "name": "Fidschi-Sprache",
    "code": "fj"
  },
  {
    "name": "Färöisch",
    "code": "fo"
  },
  {
    "name": "Französisch",
    "code": "fr"
  },
  {
    "name": "Friesisch",
    "code": "fy"
  },
  {
    "name": "Irisch",
    "code": "ga"
  },
  {
    "name": "Gälisch-Schottisch",
    "code": "gd"
  },
  {
    "name": "Gälisch-Schottisch",
    "code": "gd"
  },
  {
    "name": "Galicisch",
    "code": "gl"
  },
  {
    "name": "Guarani-Sprache",
    "code": "gn"
  },
  {
    "name": "Gujarati-Sprache",
    "code": "gu"
  },
  {
    "name": "Manxs",
    "code": "gv"
  },
  {
    "name": "Haussa-Sprache",
    "code": "ha"
  },
  {
    "name": "Hebräisch",
    "code": "he"
  },
  {
    "name": "Hindi",
    "code": "hi"
  },
  {
    "name": "Hiri-Motu",
    "code": "ho"
  },
  {
    "name": "Kroatisch",
    "code": "hr"
  },
  {
    "name": "Haitien; Frankokreolisch",
    "code": "ht"
  },
  {
    "name": "Ungarisch",
    "code": "hu"
  },
  {
    "name": "Armenisch",
    "code": "hy"
  },
  {
    "name": "Herero-Sprache",
    "code": "hz"
  },
  {
    "name": "Interlingua",
    "code": "ia"
  },
  {
    "name": "Bahasa Indonesia",
    "code": "id"
  },
  {
    "name": "Interlingue",
    "code": "ie"
  },
  {
    "name": "Ibo-Sprache",
    "code": "ig"
  },
  {
    "name": "Nosu",
    "code": "ii"
  },
  {
    "name": "Inupik",
    "code": "ik"
  },
  {
    "name": "Sinohoan",
    "code": "io"
  },
  {
    "name": "Isländisch",
    "code": "is"
  },
  {
    "name": "Italienisch",
    "code": "it"
  },
  {
    "name": "Inuktitut",
    "code": "iu"
  },
  {
    "name": "Japanisch",
    "code": "ja"
  },
  {
    "name": "Javanisch",
    "code": "jv"
  },
  {
    "name": "Georgisch",
    "code": "ka"
  },
  {
    "name": "Kongo-Sprache",
    "code": "kg"
  },
  {
    "name": "Kikuyu-Sprache",
    "code": "ki"
  },
  {
    "name": "Kikuyu-Sprache",
    "code": "ki"
  },
  {
    "name": "Kwanyama-Sprache",
    "code": "kj"
  },
  {
    "name": "Kwanyama-Sprache",
    "code": "kj"
  },
  {
    "name": "Kasachisch",
    "code": "kk"
  },
  {
    "name": "Grönländisch",
    "code": "kl"
  },
  {
    "name": "Grönländisch",
    "code": "kl"
  },
  {
    "name": "Kambodschanisch",
    "code": "km"
  },
  {
    "name": "Kannada",
    "code": "kn"
  },
  {
    "name": "Koreanisch",
    "code": "ko"
  },
  {
    "name": "Kanuri-Sprache",
    "code": "kr"
  },
  {
    "name": "Kaschmiri",
    "code": "ks"
  },
  {
    "name": "Kurdisch",
    "code": "ku"
  },
  {
    "name": "Komi-Sprache",
    "code": "kv"
  },
  {
    "name": "Kornisch",
    "code": "kw"
  },
  {
    "name": "Kirgisisch",
    "code": "ky"
  },
  {
    "name": "Latein",
    "code": "la"
  },
  {
    "name": "Luxemburgisch",
    "code": "lb"
  },
  {
    "name": "Luxemburgisch",
    "code": "lb"
  },
  {
    "name": "Ganda-Sprache",
    "code": "lg"
  },
  {
    "name": "Limburgisch",
    "code": "li"
  },
  {
    "name": "Lingala",
    "code": "ln"
  },
  {
    "name": "Laotisch",
    "code": "lo"
  },
  {
    "name": "Litauisch",
    "code": "lt"
  },
  {
    "name": "Luba-Katanga-Sprache",
    "code": "lu"
  },
  {
    "name": "Lettisch",
    "code": "lv"
  },
  {
    "name": "Malagassi-Sprache",
    "code": "mg"
  },
  {
    "name": "Marschallesisch",
    "code": "mh"
  },
  {
    "name": "Maori-Sprache",
    "code": "mi"
  },
  {
    "name": "Makedonisch",
    "code": "mk"
  },
  {
    "name": "Malayalam",
    "code": "ml"
  },
  {
    "name": "Mongolisch",
    "code": "mn"
  },
  {
    "name": "Moldauisch",
    "code": "mo"
  },
  {
    "name": "Marathi",
    "code": "mr"
  },
  {
    "name": "Malaiisch",
    "code": "ms"
  },
  {
    "name": "Maltesisch",
    "code": "mt"
  },
  {
    "name": "Burmesisch; Birmanisch",
    "code": "my"
  },
  {
    "name": "Nauruanisch",
    "code": "na"
  },
  {
    "name": "Bokmål",
    "code": "nb"
  },
  {
    "name": "Bokmål",
    "code": "nb"
  },
  {
    "name": "Ndebele-Sprache",
    "code": "nd"
  },
  {
    "name": "Ndebele-Sprache",
    "code": "nd"
  },
  {
    "name": "Nepali",
    "code": "ne"
  },
  {
    "name": "Ndonga",
    "code": "ng"
  },
  {
    "name": "Niederländisch",
    "code": "nl"
  },
  {
    "name": "Niederländisch",
    "code": "nl"
  },
  {
    "name": "Nynorsk",
    "code": "nn"
  },
  {
    "name": "Nynorsk",
    "code": "nn"
  },
  {
    "name": "Norwegisch",
    "code": "no"
  },
  {
    "name": "Ndbele-Sprache",
    "code": "nr"
  },
  {
    "name": "Ndebele-Sprache",
    "code": "nr"
  },
  {
    "name": "Navajo-Sprache",
    "code": "nv"
  },
  {
    "name": "Nyanja-Sprache",
    "code": "ny"
  },
  {
    "name": "Nyanja-Sprache",
    "code": "ny"
  },
  {
    "name": "Nyanja-Sprache",
    "code": "ny"
  },
  {
    "name": "Okzitanisch",
    "code": "oc"
  },
  {
    "name": "Okzitanisch",
    "code": "oc"
  },
  {
    "name": "Ojibwa-Sprache",
    "code": "oj"
  },
  {
    "name": "Galla-Sprache",
    "code": "om"
  },
  {
    "name": "Oriya-Sprache",
    "code": "or"
  },
  {
    "name": "Ossetisch",
    "code": "os"
  },
  {
    "name": "Pandschabi-Sprache",
    "code": "pa"
  },
  {
    "name": "Pandschabi-Sprache",
    "code": "pa"
  },
  {
    "name": "Pali",
    "code": "pi"
  },
  {
    "name": "Polnisch",
    "code": "pl"
  },
  {
    "name": "Paschtu",
    "code": "ps"
  },
  {
    "name": "Portugiesisch",
    "code": "pt"
  },
  {
    "name": "Quechua-Sprache",
    "code": "qu"
  },
  {
    "name": "Rätoromanisch; Rumantsch",
    "code": "rm"
  },
  {
    "name": "Rundi-Sprache",
    "code": "rn"
  },
  {
    "name": "Rumänisch",
    "code": "ro"
  },
  {
    "name": "Russisch",
    "code": "ru"
  },
  {
    "name": "Rwanda-Sprache",
    "code": "rw"
  },
  {
    "name": "Sanskrit",
    "code": "sa"
  },
  {
    "name": "Sardisch",
    "code": "sc"
  },
  {
    "name": "Sindhi-Sprache",
    "code": "sd"
  },
  {
    "name": "Nordsaamisch",
    "code": "se"
  },
  {
    "name": "Sango-Sprache",
    "code": "sg"
  },
  {
    "name": "Singhalesisch",
    "code": "si"
  },
  {
    "name": "Slowakisch",
    "code": "sk"
  },
  {
    "name": "Slowenisch",
    "code": "sl"
  },
  {
    "name": "Samoanisch",
    "code": "sm"
  },
  {
    "name": "Schona-Sprache",
    "code": "sn"
  },
  {
    "name": "Somali",
    "code": "so"
  },
  {
    "name": "Albanisch",
    "code": "sq"
  },
  {
    "name": "Serbisch",
    "code": "sr"
  },
  {
    "name": "Swasi-Sprache",
    "code": "ss"
  },
  {
    "name": "Süd-Sotho-Sprache",
    "code": "st"
  },
  {
    "name": "Sundanesisch",
    "code": "su"
  },
  {
    "name": "Schwedisch",
    "code": "sv"
  },
  {
    "name": "Swahili",
    "code": "sw"
  },
  {
    "name": "Tamil",
    "code": "ta"
  },
  {
    "name": "Telugu-Sprache",
    "code": "te"
  },
  {
    "name": "Tadschikisch",
    "code": "tg"
  },
  {
    "name": "Thailändisch",
    "code": "th"
  },
  {
    "name": "Tigrinja-Sprache",
    "code": "ti"
  },
  {
    "name": "Turkmenisch",
    "code": "tk"
  },
  {
    "name": "Tagalog",
    "code": "tl"
  },
  {
    "name": "Tswana-Sprache",
    "code": "tn"
  },
  {
    "name": "Tongaisch",
    "code": "to"
  },
  {
    "name": "Türkisch",
    "code": "tr"
  },
  {
    "name": "Tsonga-Sprache",
    "code": "ts"
  },
  {
    "name": "Tatarisch",
    "code": "tt"
  },
  {
    "name": "Twi-Sprache",
    "code": "tw"
  },
  {
    "name": "Tahitisch",
    "code": "ty"
  },
  {
    "name": "Uigurisch",
    "code": "ug"
  },
  {
    "name": "Uigurisch",
    "code": "ug"
  },
  {
    "name": "Ukrainisch",
    "code": "uk"
  },
  {
    "name": "Urdu",
    "code": "ur"
  },
  {
    "name": "Usbekisch",
    "code": "uz"
  },
  {
    "name": "Venda-Sprache",
    "code": "ve"
  },
  {
    "name": "Vietnamesisch",
    "code": "vi"
  },
  {
    "name": "Volapük",
    "code": "vo"
  },
  {
    "name": "Wallonisch",
    "code": "wa"
  },
  {
    "name": "Wolof-Sprache",
    "code": "wo"
  },
  {
    "name": "Xhosa-Sprache",
    "code": "xh"
  },
  {
    "name": "Jiddisch",
    "code": "yi"
  },
  {
    "name": "Yoruba-Sprache",
    "code": "yo"
  },
  {
    "name": "Zhuang",
    "code": "za"
  },
  {
    "name": "Zhuang",
    "code": "za"
  },
  {
    "name": "Chinesisch",
    "code": "zh"
  },
  {
    "name": "Zulu-Sprache",
    "code": "zu"
  }
]
