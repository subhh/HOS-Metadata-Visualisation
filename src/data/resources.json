{
  "Article": "Artikel",
  "Dissertation": "Dissertation",
  "Book": "Buch",
  "Book Part": "Buch (Auszug)",
  "Thesis": "Thesis",
  "Digital": "Digital",
  "Working Paper": "Diskussionspapier",
  "Technical Report": "Technischer Bericht",
  "Conference Object": "Konferenzobjekt",
  "Other": "Andere",
  "Research Paper": "Wissenschaftliche Publikation",
  "Proceedings Part": "Tagungsband (Auszug)",
  "Proceedings": "Tagungsband",
  "Preprint": "Vorabdruck",
  "Dataset": "Daten",
  "Interactive Resource": "interaktive Ressource",
  "Manual": "Handbuch",
  "Diagram": "Diagramm",
  "Learning Object": "Lernmaterial"
}
