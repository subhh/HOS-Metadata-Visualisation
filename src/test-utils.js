import React, { useReducer } from 'react'
import { render } from '@testing-library/react'
import { ThemeProvider } from 'styled-components'
import { MemoryRouter } from 'react-router-dom'
import { StateContext, DispatchContext, reducer, initialState } from './state'
import theme from './theme'

const AllTheProviders = ({ children }) => {
  const [state, dispatch] = useReducer(reducer, initialState)

  return (
    <ThemeProvider theme={theme}>
      <StateContext.Provider value={state}>
        <DispatchContext.Provider value={dispatch}>
          {children}
        </DispatchContext.Provider>
      </StateContext.Provider>
    </ThemeProvider>
  )
}

const WithRouter = ({ children }) => (
  <MemoryRouter>
    <AllTheProviders>{children}</AllTheProviders>
  </MemoryRouter>
)

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options })

export const customRenderWithRouter = (ui, options) =>
  render(ui, {
    wrapper: WithRouter,
    ...options,
  })

// re-export everything
export * from '@testing-library/react'

// override render method
export { customRender as render }

export const mockAsync = data => {
  jest.spyOn(global, 'fetch').mockImplementation(url => {
    return Promise.resolve({
      json: () => Promise.resolve(data),
    })
  })
}

export const restoreAsyncMock = () => {
  global.fetch.mockRestore()
}
