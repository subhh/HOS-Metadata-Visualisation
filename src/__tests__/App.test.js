import React from 'react'
import { MemoryRouter } from 'react-router-dom'
import { render, mockAsync, restoreAsyncMock, wait } from 'test-utils'
import { AppWithoutProvider } from 'App'
import data from '__mocks__/solrLocationsMock'

it('Route /map should match snapshot', async () => {
  mockAsync(data)
  const { container } = render(
    <MemoryRouter initialEntries={['/map']}>
      <AppWithoutProvider />
    </MemoryRouter>
  )
  await wait(() => expect(container).toBeTruthy())
  expect(container).toMatchSnapshot()
  restoreAsyncMock()
})

it('Route /graph should match snapshot', () => {
  const { container } = render(
    <MemoryRouter initialEntries={['/graph']}>
      <AppWithoutProvider />
    </MemoryRouter>
  )
  expect(container).toMatchSnapshot()
})

it('Route /graph/explore should match snapshot', () => {
  const { container } = render(
    <MemoryRouter initialEntries={['/graph/explore?q=test']}>
      <AppWithoutProvider />
    </MemoryRouter>
  )
  expect(container).toMatchSnapshot()
})
