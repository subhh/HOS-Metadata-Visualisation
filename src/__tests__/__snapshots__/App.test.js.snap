// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route /graph should match snapshot 1`] = `
<div>
  <div
    class="sc-caSCKo OLMic"
  >
    <div
      class="sc-caSCKo OLMic"
    >
      <div
        class="sc-bdVaJa sc-jDwBTQ wpHBS"
        height="307"
        width="1024"
      />
      <a
        class="sc-Rmtcm gwGgYM sc-gisBJw kTmEgj"
        href="/graph/explore"
      >
        <button
          class="sc-bdVaJa sc-EHOje sc-bRBYWo krxYEN"
          color="blueMediumDark"
          font-size="inherit"
          font-weight="bold"
        >
          Themen erkunden
        </button>
      </a>
    </div>
    <div
      class="sc-bdVaJa sc-jhAzac ikxoXM"
      font-size="1"
    >
      <div
        class="sc-bdVaJa sc-fBuWsC sc-dVhcbM ftgZZc"
        data-testid="toggle"
        height="40"
        width="40"
      >
        <svg
          class="sc-fAjcbJ loiswx"
          fill="blueMediumDark"
        >
          help.svg
        </svg>
      </div>
    </div>
    <div
      class="sc-kjoXOD NPhyn"
    >
      <div
        class="sc-bdVaJa Search__SearchContainer-sc-18xh3im-0 hkYogB"
      >
        <div
          class="sc-bdVaJa Search__IconContainer-sc-18xh3im-2 Search__SearchIconContainer-sc-18xh3im-3 cBQBHZ"
          data-testid="submit"
          height="36"
          width="36"
        >
          <svg
            class="Search___StyledSearchSVG-sc-18xh3im-5 bSZqyx"
          >
            search.svg
          </svg>
        </div>
        <input
          class="Search__Input-sc-18xh3im-1 ShqGu"
          color="blueMediumDark"
          data-testid="input"
          font-size="1"
          height="36"
          opacity="0"
          placeholder="Suchbegriff eingeben"
          size="20"
          type="text"
          value=""
          width="36px"
        />
      </div>
    </div>
    <div
      class="sc-hzDkRC cyrhHJ sc-cHGsZl cydaOB"
    >
      <a
        class="sc-Rmtcm gwGgYM"
        href="/map"
      >
        <button
          class="sc-bdVaJa sc-EHOje sc-bRBYWo gDgYoz"
          color="blueMediumDark"
          font-size="inherit"
          font-weight="bold"
        >
          Einrichtungen
        </button>
      </a>
      <a
        aria-current="page"
        class="sc-Rmtcm gwGgYM active"
        href="/graph"
      >
        <button
          class="sc-bdVaJa sc-EHOje sc-bRBYWo gDgYoz"
          color="blueMediumDark"
          font-size="inherit"
          font-weight="bold"
        >
          Kollektionen
        </button>
      </a>
    </div>
  </div>
  <div
    class="sc-bdVaJa iHZvIS"
    style="opacity: 0;"
  >
    <div
      class="sc-bdVaJa sc-bwzfXH cnoQWm"
    >
      <div
        class="sc-bdVaJa sc-bwzfXH cPJRUX"
        width="1,1,1,1,0.75,1"
      >
        <div
          class="sc-bdVaJa sc-htpNat sc-kkGfuU isncOJ"
          color="blueDark"
        >
          Lade Statistiken ...
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Route /graph/explore should match snapshot 1`] = `
<div>
  <div
    class="sc-caSCKo OLMic"
  >
    <div
      class="sc-caSCKo OLMic"
    >
      <div
        class="sc-bdVaJa sc-jDwBTQ wpHBS"
        height="307"
        width="1024"
      />
      <a
        class="sc-Rmtcm gwGgYM sc-gisBJw kTmEgj"
        href="/graph"
      >
        <button
          class="sc-bdVaJa sc-EHOje sc-bRBYWo krxYEN"
          color="blueMediumDark"
          font-size="inherit"
          font-weight="bold"
        >
          zur Ãœbersicht
        </button>
      </a>
    </div>
    <div
      class="sc-bdVaJa sc-jhAzac ikxoXM"
      font-size="1"
    >
      <div
        class="sc-bdVaJa sc-fBuWsC sc-dVhcbM ftgZZc"
        data-testid="toggle"
        height="40"
        width="40"
      >
        <svg
          class="sc-fAjcbJ loiswx"
          fill="blueMediumDark"
        >
          help.svg
        </svg>
      </div>
    </div>
    <div
      class="sc-kjoXOD NPhyn"
    >
      <div
        class="sc-bdVaJa Search__SearchContainer-sc-18xh3im-0 hkYogB"
      >
        <div
          class="sc-bdVaJa Search__IconContainer-sc-18xh3im-2 Search__SearchIconContainer-sc-18xh3im-3 cBQBHZ"
          data-testid="submit"
          height="36"
          width="36"
        >
          <svg
            class="Search___StyledSearchSVG-sc-18xh3im-5 bSZqyx"
          >
            search.svg
          </svg>
        </div>
        <input
          class="Search__Input-sc-18xh3im-1 ShqGu"
          color="blueMediumDark"
          data-testid="input"
          font-size="1"
          height="36"
          opacity="0"
          placeholder="Suchbegriff eingeben"
          size="20"
          type="text"
          value=""
          width="36px"
        />
      </div>
    </div>
    <div
      class="sc-hzDkRC cyrhHJ sc-cHGsZl cydaOB"
    >
      <a
        class="sc-Rmtcm gwGgYM"
        href="/map?q=test"
      >
        <button
          class="sc-bdVaJa sc-EHOje sc-bRBYWo gDgYoz"
          color="blueMediumDark"
          font-size="inherit"
          font-weight="bold"
        >
          Einrichtungen
        </button>
      </a>
      <a
        aria-current="page"
        class="sc-Rmtcm gwGgYM active"
        href="/graph?q=test"
      >
        <button
          class="sc-bdVaJa sc-EHOje sc-bRBYWo gDgYoz"
          color="blueMediumDark"
          font-size="inherit"
          font-weight="bold"
        >
          Kollektionen
        </button>
      </a>
    </div>
  </div>
  <div
    class="sc-bdVaJa iHZvIS"
    style="opacity: 0;"
  >
    <div
      class="sc-bdVaJa sc-bwzfXH cnoQWm"
    >
      <div
        class="sc-bdVaJa sc-bwzfXH cPJRUX"
        width="1,1,1,1,0.75,1"
      >
        <div
          class="sc-bdVaJa sc-htpNat sc-kkGfuU isncOJ"
          color="blueDark"
        >
          Lade Statistiken ...
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Route /map should match snapshot 1`] = `
<div>
  <div
    class="sc-caSCKo OLMic"
  >
    <div
      data-mock="fake-map"
    />
    <div
      class="sc-bdVaJa sc-jhAzac ikxoXM"
      font-size="1"
    >
      <div
        class="sc-bdVaJa sc-fBuWsC sc-dVhcbM ftgZZc"
        data-testid="toggle"
        height="40"
        width="40"
      >
        <svg
          class="sc-fAjcbJ loiswx"
          fill="blueMediumDark"
        >
          help.svg
        </svg>
      </div>
    </div>
    <div
      class="sc-kjoXOD NPhyn"
    >
      <div
        class="sc-bdVaJa Search__SearchContainer-sc-18xh3im-0 hkYogB"
      >
        <div
          class="sc-bdVaJa Search__IconContainer-sc-18xh3im-2 Search__SearchIconContainer-sc-18xh3im-3 cBQBHZ"
          data-testid="submit"
          height="36"
          width="36"
        >
          <svg
            class="Search___StyledSearchSVG-sc-18xh3im-5 bSZqyx"
          >
            search.svg
          </svg>
        </div>
        <input
          class="Search__Input-sc-18xh3im-1 ShqGu"
          color="blueMediumDark"
          data-testid="input"
          font-size="1"
          height="36"
          opacity="0"
          placeholder="Suchbegriff eingeben"
          size="20"
          type="text"
          value=""
          width="36px"
        />
      </div>
    </div>
    <div
      class="sc-hzDkRC cyrhHJ sc-cHGsZl cydaOB"
    >
      <a
        aria-current="page"
        class="sc-Rmtcm gwGgYM active"
        href="/map"
      >
        <button
          class="sc-bdVaJa sc-EHOje sc-bRBYWo gDgYoz"
          color="blueMediumDark"
          font-size="inherit"
          font-weight="bold"
        >
          Einrichtungen
        </button>
      </a>
      <a
        class="sc-Rmtcm gwGgYM"
        href="/graph"
      >
        <button
          class="sc-bdVaJa sc-EHOje sc-bRBYWo gDgYoz"
          color="blueMediumDark"
          font-size="inherit"
          font-weight="bold"
        >
          Kollektionen
        </button>
      </a>
    </div>
  </div>
  <div
    class="sc-bdVaJa iHZvIS"
    style="opacity: 0;"
  >
    <div
      class="sc-bdVaJa sc-bwzfXH cnoQWm"
    >
      <div
        class="sc-bdVaJa sc-bwzfXH cPJRUX"
        width="1,1,1,1,0.75,1"
      >
        <div
          class="sc-bdVaJa sc-htoDjs bZvMhP"
        >
          <h2
            class="sc-bdVaJa sc-htpNat sc-bxivhb sc-dnqmqq fCKnkB"
            color="blueDark"
            font-size="2"
            font-weight="500"
          >
            Lizenzstatus
          </h2>
          <div
            class="sc-bdVaJa sc-eNQAEJ dChLpc"
          >
            <div
              class="sc-bdVaJa sc-jzJRlG kHObkz"
              data-testid="grid"
              size="10"
            />
          </div>
          <ul
            class="sc-ckVGcZ eHdLWN"
          />
        </div>
        <div
          class="sc-bdVaJa sc-htoDjs bZvMhP"
        >
          <h2
            class="sc-bdVaJa sc-htpNat sc-bxivhb sc-dnqmqq fCKnkB"
            color="blueDark"
            font-size="2"
            font-weight="500"
          >
            Erscheinungsjahr
          </h2>
          <div
            class="sc-bdVaJa sc-kAzzGY csheVq"
            height="190"
            width="76"
          >
            <div
              class="sc-bdVaJa sc-chPdSV ecOJGY"
              y="79"
            >
              <div
                class="sc-bdVaJa sc-kgoBCf loeOHy"
                width="28"
              />
              <div
                class="sc-bdVaJa sc-htpNat sc-kpOJdX gfIBzX"
                color="grey"
                font-size="0"
                width="40"
              >
                0
              </div>
            </div>
          </div>
          <ul
            class="sc-ckVGcZ dOyuhG"
          >
            <li
              class="sc-jKJlTe hqaXqp"
              font-size="0"
            >
              Erfassung abgeschlossen
            </li>
            <li
              class="sc-jKJlTe ItYRm"
              font-size="0"
            >
              Erfassung in Arbeit
            </li>
          </ul>
        </div>
        <div
          class="sc-bdVaJa sc-htoDjs bZvMhP"
        >
          <h2
            class="sc-bdVaJa sc-htpNat sc-bxivhb sc-dnqmqq fCKnkB"
            color="blueDark"
            font-size="2"
            font-weight="500"
          >
            Sprache
          </h2>
          <div
            class="sc-kEYyzF kGRPgZ"
            height="238"
          >
            <table
              class="sc-iwsKbI XZnfg"
            >
              <tbody />
            </table>
          </div>
        </div>
        <div
          class="sc-bdVaJa sc-htoDjs bZvMhP"
        >
          <h2
            class="sc-bdVaJa sc-htpNat sc-bxivhb sc-dnqmqq fCKnkB"
            color="blueDark"
            font-size="2"
            font-weight="500"
          >
            Publikationsform
          </h2>
          <div
            class="sc-kEYyzF kGRPgZ"
            height="238"
          >
            <table
              class="sc-iwsKbI XZnfg"
            >
              <tbody />
            </table>
          </div>
        </div>
        <div
          class="sc-bdVaJa sc-htoDjs bZvMhP"
        >
          <h2
            class="sc-bdVaJa sc-htpNat sc-bxivhb sc-dnqmqq fCKnkB"
            color="blueDark"
            font-size="2"
            font-weight="500"
          >
            Thema
          </h2>
          <div
            class="sc-bdVaJa sc-eNQAEJ dChLpc"
          >
            <div
              class="sc-bdVaJa sc-jzJRlG kHObkz"
              data-testid="grid"
              size="10"
            />
          </div>
          <ul
            class="sc-ckVGcZ eHdLWN"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
